src/main/kotlin/net/atos/zac/app/klant/model/personen/RestPersoonIndicaties.kt

```diff
@@ -0,0 +1,10 @@
+package net.atos.zac.app.klant.model.personen
+
+enum class RestPersoonIndicaties {
+    IN_ONDERZOEK,
+    GEHEIMHOUDING_OP_PERSOONSGEGEVENS,
+    OVERLEDEN,
+    OPSCHORTING_BIJHOUDING,
```

Comment:
I am not sure where this value comes from, but it reads a bit odd. Has this been checked with Camiel and/or Karin ?

Reply:
O wow, does this come from the BRP model? Strange name.